canonical/cloud-init
# [tools/migrate-lp-user-to-github] Rename master branch to main

Rename master branch to main

Blacken the script.

## Test Steps
Run tools/migrate-lp-user-to-github, see it doesn't fail with a missing 'master' branch.

## Checklist:
